docs/advanced-guide/key-value-store/page.md

```diff
@@ -0,0 +1,80 @@
+# Key Value Store
```

Comment:
I'm thinking about it

Shouldn't Redis storage be considered as a KVStore

Reply:
It would also support it, it would be configurable, user can use any kvstore of their choice

Reply:
My point is more the fact, gofr already supports Redis


https://pkg.go.dev/gofr.dev/pkg/gofr/datasource/redis

So my point was about the fact Redis code could be refactored to rely on KVStore interface maybe

Plus the fact Redis, could be documented here in this Markdown section.

Because right now it says somehow "badger only for now, other maybe later", while Redis is already there.

Please note, I might have misunderstood something.

Reply:
Do @vipul-rawat @srijan-27 @aryanmehrotra

What are your thoughts?